missing roslaunch dependency

I believe since CMakeLists.txt find_packages roslaunch, it should be listed in package.xml as build_depend

Yes, without the dependency, I get
```
CMake Error at /home/kruset/work/ros/jade_sources/devel/share/catkin/cmake/catkinConfig.cmake:75 (find_package):
  Could not find a package configuration file provided by "roslaunch" with
  any of the following names:

    roslaunchConfig.cmake
    roslaunch-config.cmake
```

and the build fails accordingly:
```
CMake Error at CMakeLists.txt:7 (roslaunch_add_file_check):
  Unknown CMake command "roslaunch_add_file_check".

```

--- a/package.xml
+++ b/package.xml
@@ -11,7 +11,7 @@
   <url>http://ros.org/wiki/urdf_tutorial</url>
 
   <buildtool_depend>catkin</buildtool_depend>
-
+  <build_depend>roslaunch</build_depend>
   <run_depend>controller_manager</run_depend>
   <run_depend>diff_drive_controller</run_depend>
   <run_depend>gazebo_ros</run_depend>

